src/index.js

```diff
@@ -384,6 +413,33 @@ class ReactNativeModal extends Component {
     }
   };
 
+  ableToScrollInSwipeDirection = () => {
```

Comment:
The bug i'm talking about above is related to this function, by the way. That i'm sure of.